### ISSUE TITLE
Multi-tenancy with custom domains

### ISSUE THREAD

**Issue by nileshtrivedi** [hometown-fork/hometown#1205]
### Pitch

Allow one instance to host multiple domains

Mastodon issue: https://github.com/mastodon/mastodon/issues/2668

### Motivation

Like email, hosting costs can be divided across multiple orgs while giving them control of their identity by domain ownership. Please see the links issue for more on this.

**Comment by dariusk** [comment]
Am I reading the original issue correctly that the idea is I could have an account physically hosted on alpha.example, which is a server I don't own but have an account on, and then configure DNS on beta.example, which is a domain I _do_ own, to act as a kind of proxy name for it? so people could follow me at `@myname@beta.example` even though the account is hosted on the alpha.example server?

**Comment by nileshtrivedi** [comment]
Yes, I think the idea is that the domain in my ID `@myname@beta.example` and domain of my actor hosted by the Mastodon/Hometown instance `alpha.example` don't have to be identical.

**Comment by robsonsobral** [comment]
Actually, you can already allow people to follow you from a different domain than the one which hosts your account. However, this isn't transparent. The domain on the profile page you always be `alpha.example`. Being more accurate, the idea is to make so simple to host multiple domains as it is to host emails.

The better approach to that is the issue.

**Comment by dariusk** [comment]
Yes, that's the key difference between this and the current `LOCAL_DOMAIN`, is that one physical host can host multiple domains.
